index.js

```diff
@@ -54,13 +54,21 @@ function inspectElements() {
 
 const addExtensionIfInstalled = (name, getPath) => {
 	const isExtensionInstalled = name => {
+		if (session.defaultSession.getAllExtensions) {
```

Comment:
Add a comment that mentions that this code-path is for Electron 9 and later. Same with the other place.